src/plonkish/compiler/cell_manager.rs

```diff
@@ -299,6 +301,48 @@ impl CellManager for MaxWidthCellManager {
             forward_signal_row
         } as u32;
 
+
+
+        let mut fixed_signal_column: usize = 0;
+        let mut fixed_signal_row: usize = 0;
+
+        for fixed_signal in unit.fixed_signals.iter() {
+            let column = if placement.columns.len() <= fixed_signal_column {
```

Comment:
@atsupontio you should add some tests, this will not work. Because placement.columns contains both advice and fixed signals, if there are both, for example when `fixed_signal_column == 0` but there are already columns that are advice the fixed column will not be created and the wrong column will be picked up